### PR TITLE
added plus icon and changed  the background color

### PR DESCRIPTION
**Pull request by Arjun8532**
Targets #2288 

### Description
added  plus icon and changed the background color of Add client

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Resources/views/index.blade.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <div class="d-none d-md-flex justify-content-between my-2">
         @include('client::menu_header')
         @can('clients.create')
-            <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add client</a>
+            <a href= "{{ route('client.create') }}" class="btn btn-success float-right"><i class="fa fa-plus"></i> Add client</a>
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/index.blade.php:8]: Why did you add the float-right class here? While it was not needed.

**Comment by Ayush8923** [comment]
Please resolve merge conflict.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
@Arjun8532 As far as I see the same code is written twice. Does it need to be written twice?

File: Modules/Client/Resources/views/index.blade.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <div class="d-none d-md-flex justify-content-between my-2">
         @include('client::menu_header')
         @can('clients.create')
-            <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add New Client</a>
+            <a href= "{{ route('client.create') }}" class="btn btn-success text-white"><i class="fa fa-plus"></i> Add new client</a>
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/index.blade.php:8]: As far as I see the same code is written twice. Does it need to be written twice?

**Review by Ayush8923 — CHANGES_REQUESTED** [review]

File: Modules/Client/Resources/views/index.blade.php [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <div class='d-md-none mb-2'>
         @can('clients.create')
             <div class="d-flex flex-row-reverse">
-                <a href= "{{ route('client.create') }}" class="btn btn-info text-white">Add client</a>
+                <a href= "{{ route('client.create') }}" class="btn btn-primary text-white">Add client</a>
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/index.blade.php:26]: Please change the name of the button here also. And also add plus icon. As done in line no. 8.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

File: Modules/Client/Resources/views/index.blade.php (hunk 3 above)
Comment (Arjun8532) [reply Modules/Client/Resources/views/index.blade.php:26]: Xamp was not functioning properly so  I  am reinstalling my portal as soon as I will be able to reset the portal
 I will make the suitable changes

**Comment by gautam071** [comment]
@Arjun8532 There are some conflicts please resolve them. 

If you don't know how. Please go through this link https://github.com/ColoredCow/portal/blob/master/docs/contributing.md#merge-conflict-

**Comment by Ayush8923** [comment]
The issue is already fixed.

**Pull request closed without merging**